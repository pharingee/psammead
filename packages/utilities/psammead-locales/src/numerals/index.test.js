import * as numerals from '.';

describe('Numerals', () => {
  const numeralSystems = Object.keys(numerals);
  numeralSystems.forEach(numeralSystem => {
    it(`should return ${numeralSystem} numerals`, () => {
      expect(numerals[numeralSystem]).toMatchSnapshot();
    });
  });
});
