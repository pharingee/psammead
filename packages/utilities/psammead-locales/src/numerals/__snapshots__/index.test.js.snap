// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Numerals should return Bengali numerals 1`] = `
Array [
  "০",
  "১",
  "২",
  "৩",
  "৪",
  "৫",
  "৬",
  "৭",
  "৮",
  "৯",
  "১০",
]
`;

exports[`Numerals should return Burmese numerals 1`] = `
Array [
  "၀",
  "၁",
  "၂",
  "၃",
  "၄",
  "၅",
  "၆",
  "၇",
  "၈",
  "၉",
  "၁၀",
]
`;

exports[`Numerals should return EasternArabic numerals 1`] = `
Array [
  "۰",
  "۱",
  "۲",
  "۳",
  "۴",
  "۵",
  "۶",
  "۷",
  "۸",
  "۹",
  "۱۰",
]
`;

exports[`Numerals should return WesternArabic numerals 1`] = `
Array [
  "0",
  "1",
  "2",
  "3",
  "4",
  "5",
  "6",
  "7",
  "8",
  "9",
  "10",
]
`;
